spectrum_utils/plot.py

```diff
@@ -286,3 +430,83 @@ def mirror(
     )
 
     return ax
+
+
+def facet(
+    spec_top: MsmsSpectrum,
+    spec_mass_errors: Optional[MsmsSpectrum] = None,
+    spec_bottom: Optional[MsmsSpectrum] = None,
+    spectrum_kws: Optional[Mapping[str, Any]] = None,
+    mass_errors_kws: Optional[Mapping[str, Any]] = None,
+    height: Optional[float] = None,
+    aspect: Optional[float] = None,
+) -> plt.Figure:
+    """
+    Plot a spectrum, and optionally mass errors, and a mirror spectrum.
+
+    Parameters
+    ----------
+    spec_top : MsmsSpectrum
+        The spectrum to be plotted on the top.
+    spec_mass_errors : Optional[MsmsSpectrum], optional
+        The spectrum for which mass errors are to be plotted in the middle.
+    spec_bottom : Optional[MsmsSpectrum], optional
+        The spectrum to be plotted on the bottom.
+    spectrum_kws : Optional[Mapping[str, Any]], optional
+        Keyword arguments for `plot.spectrum` for the top and bottom spectra.
+    mass_errors_kws : Optional[Mapping[str, Any]], optional
+        Keyword arguments for `plot.mass_errors`.
+    height : Optional[float], optional
+        The height of the figure in inches.
+    aspect : Optional[float], optional
```

Comment:
**todo:** Ultimately the result is the same, but I find directly specifying the width more straightforward. This also matches the standard matplotlib API better.